examples/imp-tests.dx

```diff
@@ -0,0 +1,15 @@
+'# `Imp` tests
+
+%passes imp
```

Comment:
I don't think we need this test. We usually avoid tests that expose pretty-printed IR because it's too sensitive to changes in either the pretty printer or the upstream passes.

Reply:
Got it! Removed.